packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -202,7 +202,7 @@ async function makeE2ETasks () {
       process.chdir(cachedCwd)
       testState = {}
       remoteGraphQLOptions = {}
-      await DataContext.waitForActiveRequestsToFlush()
```

Comment:
What are the implications of this?

Reply:
It seems to be like @cacieprins said, this is setup like a singleton & isn't being reset between cy & cy runs. I have not observed any issues with resetting this. The rest is handled on line 209 with `cy.reinitializeCypress()`.